### ISSUE TITLE
VSCode status bar display is weird in the latest (2024.44.0)

### ISSUE THREAD

**Issue by ryokaji** [astral-sh/ruff-vscode#605]
## What happens
When `2024.44.0` is installed, the Status bar display in the default (v0.6.4) looks like hunging up as follows.
![image](https://github.com/user-attachments/assets/3c5146dd-ea99-4a1f-a4d2-780587691b3a)

This does not occur if Ruff v0.5.7 is explicitly specified in the settings.
![image](https://github.com/user-attachments/assets/aba366b3-feef-4a47-93b5-e9a536abb854)

*note: This issue is not reporting a bug, the extension looks working fine.

## System Info
VSCode version: 1.91.1
Python version: 3.11.2
Host OS:
* Microsoft Windows 10 Home 22H2 19045.4780
* macOS Sonoma 14.5

## Project structure
The following stuff is minimum configuration to reproduce the issue.
### Settings
```json5
{
    // Configuration to explicitly specify v0.5.7
    // Assuming that a venv has been created in ~/poc.
    "ruff.path": [
        "/home/node/poc/bin/ruff"
    ]
}
```

### Others

files:
```
.
├── hello.py
└── .vscode
    └── settings.json
```

hello.py
```python
print("hello")

```

**Comment by MichaReiser** [comment]
Thanks. I can reproduce this by just opening a python project.
